mythril/laser/ethereum/instructions.py

```diff
@@ -1871,20 +1899,70 @@ def call_(self, global_state: GlobalState) -> List[GlobalState]:
             callee_account=callee_account,
             call_data=call_data,
             call_value=value,
+            static=environment.static,
         )
-        raise TransactionStartSignal(transaction, self.op_code)
+        raise TransactionStartSignal(transaction, self.op_code, global_state)
 
     @StateTransition()
     def call_post(self, global_state: GlobalState) -> List[GlobalState]:
         """
 
+        :param global_state:
+        :return:
+        """
+
+        return self.post_handler(global_state, function_name="call")
+
+    @StateTransition()
```

Comment:
This is also a state mutating instruction according to the yellowpaper

Reply:
But the EIP calls it some exception https://github.com/ethereum/EIPs/blob/master/EIPS/eip-214.md .

Reply:
I've checked the geth and parity implementations an the EIP is indeed what is implemented. 
So the status quo is good 👍